modules/custom/az_news/config/install/views.view.az_news.yml

```diff
@@ -84,7 +87,7 @@ display:
         type: 'entity:node'
         options:
           relationship: none
-          view_mode: az_media_list
+          view_mode: az_feature
```

Comment:
```suggestion
          view_mode: az_media_list
```
If not

Reply:
I don't know if you have already done this, but we could also consider resaving all of the displays for the row settings marking them as "This display only" on save.

Reply:
I did that for sure on paged row view and feature view! I believe the others were already like that, but I can double check.

Reply:
<img width="1074" alt="image" src="https://user-images.githubusercontent.com/1023167/168176916-8471a9ec-7379-43f5-b044-8b2df60f19ca.png">